specification/security/resource-manager/Microsoft.Security/preview/2022-03-01-preview/vmScanners.json

```diff
@@ -0,0 +1,259 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
```

Comment:
Make sure to go over the files that you added and replace Azure Security Center with Defender for Cloud